orm.go

```diff
@@ -453,3 +469,19 @@ const (
 	TimeQuantumMonthDayHour     TimeQuantum = "MDH"
 	TimeQuantumYearMonthDayHour TimeQuantum = "YMDH"
 )
+
+type CacheType string
```

Comment:
The data encapsulation argument doesn't hold water - user code would look like `CacheType("blah")` which would just be a compile error if CacheType was changed to int. If the user code was using the constants, then it would continue working fine either way.

If we really wanted to encapsulate, we could make `cacheType` an unexported struct type with 1 string field, and only export variables of that type, so that user code is forced to use the variables since it cannot construct arbitrary values of that type. I think that'd be overkill though.

In any case, if you want to keep it as a separate type, I disagree, but I'll support it.